src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
```

Comment:
Consider removing the empty line at the beginning of the class definition to follow best practices.